lib/VCMI_Lib.cpp

```diff
@@ -52,7 +52,8 @@ DLL_LINKAGE void preinitDLL(CConsoleHandler * Console, bool onlyEssential, bool
 	console = Console;
 	VLC = new LibClasses();
 	VLC->loadFilesystem(extractArchives);
-	settings.init();
+	settings.init("config/settings.json", "vcmi:settings");
+	persistentStorage.init("config/persistentStorage.json", "");

```

Comment:
what the purpose for that? is it user data? If so, maybe consider another name?

Reply:
I think it's okay.

It's a generic data storage for persistent things (campaign results, highscores, etc...). Does not necessarily have to be userdata.